.gitignore

```diff
@@ -17,3 +17,6 @@ config.json
 *.pyc
 *.bak
 /src
+# pixi environments
```

Comment:
Even though pixi is not supported. I think this is fine since people may want to use pixi, like @hameerabbasi  himself. :) And excluding developer files is always wanted.

Reply:
I added a few more excludes in that case.